build-tools/installers/create-installers.targets

```diff
@@ -102,8 +102,14 @@
   </ItemGroup>
   <ItemGroup>
     <_MSBuildFiles Include="$(MSBuildSrcDir)\android-support-multidex.jar" />
-    <_MSBuildFiles Include="$(MSBuildSrcDir)\apkdiff.exe" />
-    <_MSBuildFiles Include="$(MSBuildSrcDir)\apkdiff.pdb" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\apkdiff\apkdiff.exe" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\apkdiff\apkdiff.pdb" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\apkdiff\Mono.Options.dll" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\apkdiff\Mono.Posix.NETStandard.dll" />
```

Comment:
I think we can remove this addition, see https://github.com/xamarin/xamarin-android/pull/4267 for context.